crates/aptos-ledger/src/lib.rs

```diff
@@ -143,17 +143,86 @@ pub fn get_app_name() -> Result<String, AptosLedgerError> {
     }
 }
 
-/// Returns the public key of your Aptos account in Ledger device
+/// Returns the next available account in Ledger starting from start_index to end_index, inclusive
+/// Note: We only allow a range of 20 for performance purpose
+///
+/// # Arguments
+///
+/// * `start_index` - The index to start fetch for accounts
+/// * `end_index` - The last index to fetch the accounts
+pub fn fetch_batch_accounts(
+    start_index: u32,
+    end_index: u32,
```

Comment:
There is a native representation of this idea already, consider taking a [Range](https://doc.rust-lang.org/std/ops/struct.Range.html) instead.